### PR TITLE
User delete feature (for admin)

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/161365399

NOTEL this PR is built on top of the rails-upgrade PR (https://github.com/AgileVentures/shf-project/pull/554).

The actual number changed files for _this_ PR are:

```
modified:   app/controllers/users_controller.rb
modified:   app/views/users/_user.html.haml
modified:   app/views/users/_users_list.html.haml
modified:   config/locales/en.yml
modified:   config/locales/sv.yml
modified:   features/manage_users.feature
```

Changes proposed in this pull request:
1. Add delete icon (garbage can) to users index view
2. Controller action
3. Cucumber test

Screenshots (Optional):
screen shot 2018-10-29 at 10 28 53 am

---

screen shot 2018-10-29 at 10 30 22 am

Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

File: app/views/users/_user.html.haml [hunk 1]
```diff
@@ -16,3 +16,7 @@
     - if expire_date
       %span{ class: expire_date_css_class(expire_date) }
         = expire_date
+  %td.delete
+    = link_to "<span class='glyphicon glyphicon-trash text-danger'>".html_safe,
+              user_path(user), method: :delete, title: "delete #{user.full_name}",
+              :data => {:confirm => "Delete #{user.full_name}?"}
```
Comment (weedySeaDragon) [inline app/views/users/_user.html.haml]: need translation for confirmation message:  `:confirm => "#{t(delete...)} #{user.full_name}?"`

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
Looks good with that 1 minor i18n change.

File: app/views/users/_user.html.haml (hunk 1 above)
Comment (patmbolger) [reply app/views/users/_user.html.haml]: @weedySeaDragon - wow, I can't believe I forgot about translation!  Thanks for catching that - fixed now.

Comment (weedySeaDragon) [reply app/views/users/_user.html.haml]: @patmbolger I bet you left that out just to give me something to do. ;-)

**Review by weedySeaDragon — APPROVED** [review]

**Pull request merged**